doc/assessment.md

```diff
@@ -77,35 +106,133 @@ An assessment is broken down in to a list of zones, like this:
 ],
 ```
 
-* Each zone appears in the given order in the assessment. Zone titles are optional and are displayed to the student if present.
+Zone Property | Type | Description
+--- | --- | ---
+`title` | string | The title of the zone. (Optional; default: none)
+`questions` | array | The list of slots for questions and question alternatives within the zone. (Optional; default: none)
+`numberChoose` | integer | Number of questions to select for each student from this zone. (Optional; default: select all)
+`maxPoints` | number | Limit on the number of points that can be earned from this zone. (Optional; default: sum of question max points)
+`bestQuestions` | integer | Only this many questions in the zone will count towards the total points (highest-point questions will count). (Optional; default: use all questions)
+
```

Comment:
Add link again to the schema specification?

```
[Format specification for zones can be found in `infoAssessment.json`](https://github.com/PrairieLearn/PrairieLearn/blob/master/schemas/schemas/infoAssessment.json)
```

Reply:
done